### ISSUE TITLE
SharedInformerFactory missing a Shutdown method

### ISSUE THREAD

**Issue by trozet** [k8snetworkplumbingwg/multi-networkpolicy#22]
The Shutdown method is useful for waiting for the informer to be stopped when running unit tests:
https://github.com/kubernetes/code-generator/commit/e4b7976d1aa480d750be54b0177e5e29790de3bb

It is missing:
https://github.com/k8snetworkplumbingwg/multi-networkpolicy/blob/master/pkg/client/informers/externalversions/internalinterfaces/factory_interfaces.go#L34

I tried updating code-gen and regenerating, but it doesnt seem to work. Not sure why...

**Comment by jcaamano** [comment]
WE have a similar situation with the NAD factory

**Comment by tssurya** [comment]
yeah hit this with the NAD factory today (I wonder if there is another issue for NAD or if we should open one...)
